tests/scripts/github-action-helper.sh

```diff
@@ -143,9 +143,14 @@ function create_cluster_prerequisites() {
   kubectl create -f crds.yaml -f common.yaml
 }
 
+function deploy_manifest_with_local_build() {
+  sed -i "s|image: rook/ceph:v1.7.4|image: rook/ceph:local-build|g" $1
```

Comment:
How about using a regex here so we don't have to update this each time we change the version number?

Reply:
Yeah makes sense, I'll fix this when porting to master. For now it will be included in the search and replace we already do in the docs and manifests for the release version.